tests/test_make_info.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python
+# -*- coding: utf-8  -*-
+"""Unit tests for make_info.py."""
+from __future__ import unicode_literals
+import unittest
+from batchupload.make_info import (
+    make_info_page,
```

Comment:
We're only importing one thing here -- I don't think it would make the line long enough to require parens.